sanket/javascript/day17/assignment3/sanketA3_Q3.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
change title

Reply:
ma'am thank you for you reply i'll change the title asap

Reply:
mention this in above comment change title for all assignments